tests/dialects/test_bigquery.py

```diff
@@ -1018,7 +1018,7 @@ def test_bigquery(self):
             write={
                 "clickhouse": "SHA1(x)",
                 "bigquery": "SHA1(x)",
-                "": "SHA(x)",
+                "": "S_H_A1_DIGEST(x)",
```

Comment:
Actually you know what, let's remove these two `"": ...` tests (here and L1030).